charts/nexus3/values.yaml

```diff
@@ -299,7 +299,7 @@ rootPassword:
   key: password
 
 config:
-  # -- If `true`, enable the configuration Job.
+  # -- Enables the configuration Job if set to `true` and `rootPassword.secret` is set.
```

Comment:
The "if true" pattern is idiomatic to all of my charts, so please follow the existing idioms.